docs/integrations/sources/facebook-marketing-migrations.md

```diff
@@ -1,5 +1,17 @@
 # Facebook Marketing Migration Guide
 
+## Upgrading to 4.0.0
+
+Streams Ads-Insights-* no longer use the action_report_time parameter.
```

Comment:
I'd want to link here to the upstream API docs. If we succeed in making this a no-op, we can also remove the below update/fix steps.

```suggestion
As of June 10, 2025, the source API no longer supports the `action_report_time` parameter. Per the official [Meta API documentation](https://developers.facebook.com/docs/marketing-api/insights#discrepancy-with-ads-manager):

> Beginning June 10, 2025, to reduce discrepancies with Meta Ads Manager, `use_unified_attribution_setting` and `action_report_time` parameters will be disregarded and API responses will mimic Ads Manager settings.

Please note that if you were using `action_report_time` in an `Ads-Insights-*` stream, this config option will be ignored and will no longer be supported going forward.

```

Reply:
Removed from migration docs